src/databricks/labs/ucx/aws/access.py

```diff
@@ -206,6 +206,8 @@ def get_roles_to_migrate(self) -> list[AWSCredentialCandidate]:
         """
         Identify the roles that need to be migrated to UC from the UC compatible roles list.
         """
+        if len(list(self._ws.storage_credentials.list())) > 200:
```

Comment:
Storage credentials do not exist within AWS, it's an Azure thing

Reply:
@JCZuurmond they do exist everywhere.

Reply:
ah, is it a Databricks thing then?

Reply:
See other comment on using `used_storage_accounts` for this check

Reply:
fixed it